app/src/components/theme.ts

```diff
@@ -10,6 +14,6 @@ export const lightTheme: Theme = {
   colors: {
     ...defaultTheme.colors,
     playAccent: '#ed6a5e',
-    speakers: ['#0f4880', '#1d781d', '#db0a5b', '#8d6708', '#d43900'],
+    speakers: { 0: '#0f4880', 1: '#1d781d', 2: '#db0a5b', 3: '#8d6708', 4: '#d43900' },
```

Comment:
I plan to write some updated theme documentation on our doc site, but just a heads up - there's a new `mergeTheme` function exported which handles the deep object merging between themes (so you don't have to do `...defaultTheme`, `...defaultTheme.colors`, etc), *and* it returns a type union of `DestinationTheme & SourceTheme` - so your custom theme interface can be defined separately

```ts
interface CustomTheme {
  colors: {
    playAccent: string;
    speakers: {
      0: string;
      1: string;
      2: string;
      3: string;
      4: string;
    };
  };
}

const customTheme: CustomTheme = {
  colors: {
    playAccent: "#ed6a5e",
    speakers: {
      0: "#0f4880",
      1: "#1d781d",
      2: "#db0a5b",
      3: "#8d6708",
      4: "#d43900",
    },
  },
};

type Theme = CustomTheme & DefaultTheme
const lightTheme: Theme = mergeTheme(defaultTheme, customTheme);
```

Just food for thought :) what you have here works either way.

Reply:
ah nice, that seems much nicer :)